code/Model/SiteTree.php

```diff
@@ -289,8 +295,29 @@ class SiteTree extends DataObject implements PermissionProvider, i18nEntityProvi
     ];
 
     private static $searchable_fields = [
-        'Title',
-        'Content',
+        // See also searchableFields() which adds `FilterClass`
+        'ClassName' => [
+            'general' => false,
+        ],
+        'LastEdited' => [
+            'general' => false,
+            'filter' => WithinRangeFilter::class,
+            'field' => DateField::class,
+        ],
+        // The below fields are excluded from the filter form but
+        // are included in the general search
+        'Title' => [
+            'field' => HiddenField::class,
```

Comment:
Would it be better to have these as LiteralField's so that no HTML renders on the frontend, rather than have a bunch of hidden fields render?

Reply:
`HiddenField` is a "data" field, but `LiteralField` isn't, which means there are different methods for getting the form fields. Changing this could result in some functionality not working as expected.

It would also require special handling to avoid the title (second argument to the constructor when scaffolding the form fields) from being displayed as the "content" of the `LiteralField`.